### PR TITLE
Configured package to use sample robot

### PR DESCRIPTION
**Pull request by robjmenna**

### CODE REVIEW

File: ros2_ws/src/warehouse_robot/launch/launch_sim.launch.py [hunk 1]
```diff
@@ -58,18 +58,18 @@ def generate_launch_description():
     diff_drive_spawner = Node(
         package="controller_manager",
         executable="spawner",
-        arguments=["diff_cont", "--controller-manager-timeout", "30"],
+        arguments=["diff_cont"],
```
Comment (RoboCD) [inline ros2_ws/src/warehouse_robot/launch/launch_sim.launch.py]: Can we leave these timeouts in? For some reason my computer needs the increased timeout
```suggestion
        arguments=["diff_cont"], "--controller-manager-timeout", "30"],
```

File: ros2_ws/src/warehouse_robot/launch/launch_sim.launch.py [hunk 2]
```diff
@@ -58,18 +58,18 @@ def generate_launch_description():
...
     )
 
     joint_broad_spawner = Node(
         package="controller_manager",
         executable="spawner",
-        arguments=["joint_broad", "--controller-manager-timeout", "30"],
+        arguments=["joint_broad"],
```
Comment (RoboCD) [inline ros2_ws/src/warehouse_robot/launch/launch_sim.launch.py]: Same with leaving this timeout in
```suggestion
        arguments=["joint_broad", "--controller-manager-timeout", "30"],
```

**Review by RoboCD — CHANGES_REQUESTED** [review]

File: ros2_ws/src/warehouse_robot/launch/launch_sim.launch.py (hunk 1 above)
Comment (robjmenna) [reply ros2_ws/src/warehouse_robot/launch/launch_sim.launch.py]: Yeah, I don't see a problem with that.

File: ros2_ws/src/warehouse_robot/launch/launch_sim.launch.py [hunk 3]
```diff
@@ -58,7 +58,7 @@ def generate_launch_description():
     diff_drive_spawner = Node(
         package="controller_manager",
         executable="spawner",
-        arguments=["diff_cont", "--controller-manager-timeout", "30"],
+        arguments=["diff_cont"], "--controller-manager-timeout", "30"],
```
Comment (RoboCD) [inline ros2_ws/src/warehouse_robot/launch/launch_sim.launch.py]: I think I copy-pasted that wrong, my fault
```suggestion
        arguments=["diff_cont", "--controller-manager-timeout", "30"],
```

**Review by RoboCD — APPROVED** [review]

**Pull request merged**